uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js

```diff
@@ -250,25 +274,45 @@ define([
         vm.error = undefined;
       };
 
-      (function () {
-        // Create an empty leave request
-        vm.leaveRequest = LeaveRequestInstance.init({
-          contact_id: contactId //resolved from directive
-        }, false);
+      /**
+       * initializes the controller on loading the dialog
+       */
+      (function initController() {
+        if (directiveOptions.leaveRequest) {
+          //manager is responding to leave request
+          vm.uiOptions.isManager = directiveOptions.leaveRequest.contact_id != directiveOptions.contactId;
```

Comment:
But you are assigning the value to `isManager` later, when you call `isManagedBy()`, so why are you are doing it again here?